<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-5V1JU1J" failures="2" tests="3" name="Surefire test" time="8.458" errors="0" timestamp="2025-07-28T18:06:07 EEST">
  <testcase classname="tests.LoginTest" name="loginWithValidData" time="0.232"/>
  <testcase classname="tests.LoginTest" name="loginWithValidData" time="0.197">
    <failure type="java.lang.AssertionError" message="expected [error] but found [Epic sadface: Username and password do not match any user in this service]">
      <![CDATA[java.lang.AssertionError: expected [error] but found [Epic sadface: Username and password do not match any user in this service]
at tests.LoginTest.loginWithValidData(LoginTest.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- loginWithValidData -->
  <testcase classname="tests.LoginTest" name="loginWithValidData" time="0.202">
    <failure type="java.lang.AssertionError" message="expected [error] but found [Epic sadface: Username and password do not match any user in this service]">
      <![CDATA[java.lang.AssertionError: expected [error] but found [Epic sadface: Username and password do not match any user in this service]
at tests.LoginTest.loginWithValidData(LoginTest.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- loginWithValidData -->
</testsuite> <!-- Surefire test -->
